### PR TITLE
Setup associations

### PR DESCRIPTION
**Pull request by LarryIVC**
## **In this pull request, I did the following:**
✔️ Added a `branch`.
✔️ Added a class `Classroom`.
✔️ Created `has-many/belongs-to` relationship between `Classroom` and `Student` classes.
✔️ Added a class `Book`.
✔️ Added a class `Rental`.
✔️ Created `many-to-many` relationships between `Person `and `Book `classes using the `Rentals `class as intermediate.
✔️ Check the linter.

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```
Comment (descholar-ceo) [inline book.rb]: - [x] We are supposed to have a method which adds a rental, something like `add_rental`

File: person.rb [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
   attr_accessor :age, :name
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
```
Comment (descholar-ceo) [inline person.rb:14]: - [x] We are supposed to have a method that adds a rental, something like `add_rental`

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @LarryIVC 🤝 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
👍🏽 The PR follows the correct git flow ✔️ 
👍🏽 The PR has linters installed and it passes the linters checks ✔️ 
👍🏽 The readme file follows the correct microverse's readme template ✔️ 

## Required Changes ♻️
_Check the comments under the review._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: book.rb (hunk 1 above)
Comment (LarryIVC) [reply book.rb]: Hi @descholar-ceo, thanks for your feedback, I do that.

File: person.rb (hunk 2 above)
Comment (LarryIVC) [reply person.rb:14]: Hi @descholar-ceo, thanks for your feedback, I do that.

**Review by OyePriscilla — APPROVED** [review]
Hi @LarryIVC 👍, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Implemented add_rentals method for `book.rb` and `person.rb` ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by LarryIVC** [comment]
Hi @OyePriscilla, thanks for your time and feedback. 🌟

**Pull request merged**